modules/oncodrivefml/1.0/etc/oncodrivefml.conf

```diff
@@ -0,0 +1,33 @@
+[genome]
+build = 'hg19'
+
+[signature]
+method = 'complement'
+
+[score]
+# score file path must be the directory containing scoe files
+file = 
+format = 'pack'
+chr = 0
+chr_prefix = ""
+pos = 1
+ref = 2
+alt = 3
+score = 5
+
+[statistic]
+method = 'amean'
+discard_mnp = False
+
+sampling = 100000
+sampling_max = 1000000
+sampling_chunk = 100
+sampling_min_obs = 10
+
+    [[indels]]
+        include = True
+        method = 'max'
+        max_consecutive = 7
+
+[settings]
+# cores = 1
```

Comment:
the config gives 4 by default, I think? - but here it says 1, so does the value in config gets used?

Reply:
It will be overwritten by the value in the config, I added more info